OpenKeychain/src/main/AndroidManifest.xml

```diff
@@ -806,7 +806,12 @@
             android:exported="false"
             android:label="@string/keyserver_sync_settings_title" />
 
-        <!-- Internal classes of the remote APIs (not exported!) -->
+        <provider
+            android:name=".remote.KeychainExternalProvider"
+            android:authorities="${applicationId}.provider.exported"
```

Comment:
what about using ``org.openintents.openpgp.provider`` as the authority to keep the package naming consistent for the API?